monai/transforms/utility/array.py

```diff
@@ -1715,3 +1716,111 @@ def __call__(self, img: NdarrayOrTensor, meta_dict: Mapping | None = None) -> Nd
         if self._do_transform:
             img = self.filter(img)
         return img
+
+
+class ApplyTransformToPoints(InvertibleTransform, Transform):
+    """
+    Transform points between image coordinates and world coordinates.
```

Comment:
I think here and in the dictionary version we should state that the point tensor should have shape (C, N, 2|3) and state what C and N dimensions represent.

Reply:
We should also state that affine matrices are 3x3 if 2D points and 4x4 if 3D.

Reply:
Thanks! Update the docstring, and for affine I only mention it should be `3x3 or 4x4` since `to_affine_nd` will modify the affine based on the input shape.
https://github.com/Project-MONAI/MONAI/blob/1a8afd18d5fb132efa26d6f73c3bf1fdbccd985d/monai/data/utils.py#L945

Reply:
`to_affine_nd` will ensure that the affine will be correct but I think it's important to mention that 3x3 and 4x4 matrices can be used and when, so that it's clear when one or the other is needed or what happens to a 4x4 when 2D points are provided. It should be clear that users can provide a 3x3 for 2D, and don't have to use a 4x4 nor add an unneeded Z dimension that's all 0.

Reply:
Got it, update the docstring!